deploy/ansible/playbooks/iroha-standalone-nodes/iroha-deploy.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
should contain comments, specifying:
* why do we need python2 as a pre-task
* what are the roles

Reply:
@neewy there is documentation for each role in `roles/<role name>/README.md`
i think there is no need to write about it twice (i can specify where role brief description could be found)